operations/customers.md

```diff
@@ -170,6 +172,9 @@ Returns all customers filtered by identifiers, emails, names and other filters.
 | `ItalianDestinationCode` | string | optional | Value of Italian destination code. |
 | `ItalianFiscalCode` | string | optional | Value of Italian fiscal code. |
 | `CompanyId` | string | optional | Unique identifier of [Company](companies.md#company) the customer is associated with. |
+| `MergeTargetId` | string | optional | `Id` of the account (Customer/Company) where the customer is linked to. |
+| `ActivityState` | string | optional | [State](#activity-state) of returned customer. |
+
```

Comment:
Suggest new wording:
| `ActivityState` | string | optional | [Activity State](#activity-state) of customer record, i.e. whether active or deleted. |